dags/veda_data_pipeline/veda_dataset_pipeline.py

```diff
@@ -18,11 +19,19 @@
     "tags": ["collection", "discovery"],
 }
 
+@task
+def extract_discovery_items(**kwargs):
+    ti = kwargs.get("ti")
+    discovery_items = ti.dag_run.conf.get("discovery_items")
+    print(discovery_items)
+    return discovery_items
+
```

Comment:
In datasets with multiple inputs for discovery, we need to pass each input separately. This seemed easier than refactoring the discovery handler to support it.

Reply:
Doesn't `subdag_discover` require more keys that what we have in `discovery_items`? For example `collection`